### DISCUSSION TITLE
getting langflow running on Windows

### DISCUSSION THREAD

**Discussion by scott------** [langflow-ai/langflow#41] [General]
after running pip install langflow and watching that complete successfully I try:  
C:\Users\Scott>langflow
'langflow' is not recognized as an internal or external command,
operable program or batch file.

I CD into the site-packages langflow_backend and run:
C:\Users\Scott>cd C:\Users\Scott\AppData\Roaming\Python\Python310\site-packages\langflow_backend

C:\Users\Scott\AppData\Roaming\Python\Python310\site-packages\langflow_backend>python main.py
INFO:     Started server process [13420]
INFO:     Waiting for application startup.
INFO:     Application startup complete.
INFO:     Uvicorn running on http://127.0.0.1:5003 (Press CTRL+C to quit)

I click on the link and the page displays:
{"detail":"Not Found"}

What am I missing here :-)

**Comment by carlosrcoelho** [comment]
Try `python -m langflow`

I actually also saw your issue was solved [here](https://github.com/logspace-ai/langflow/issues/42).

**Comment by ogabrielluiz** [comment]
Hey, Scott! 
So, `python main.py` just runs the backend server.

You have to use the `langflow` or, as @CarlosRodrigoCoelho said, `python -m langflow`.

It seems you are running an outdated version of LangFlow, maybe try running `pip install langflow -U` and you should be good to go.
We will release a new version any moment now, btw.

**Comment by scott------** [comment]
Yes I'm all set.  'python -m langflow' did the trick
